src/lib/map.svelte

```diff
@@ -64,6 +64,16 @@
 	function handleMouseLeave() {
 		map.getCanvas().style.cursor = '';
 	}
+	let searchTerm = '';
+
+	function clearSearch(): void {
```

Comment:
Can we make the clearSearch function also place the cursor/focus into the input so the user can immediately begin typing a new term instead of having to click on the input again?

Reply:
Sure, thanks for pointing that out